system-parachains/people/people-kusama/src/identity_ops.rs

```diff
@@ -0,0 +1,182 @@
+	use frame_system::pallet_prelude::*;
+	use pallet_identity::Judgement;
+
+	type IdentityPallet = pallet_identity::Pallet<Runtime>;
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(PhantomData<T>);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	#[pallet::error]
+	pub enum Error<T> {
+		/// No judgement to clear.
+		NotFound,
+	}
+
+	pub(crate) type Identity = (
+		pallet_identity::Registration<
+			<Runtime as pallet_balances::Config>::Balance,
+			<Runtime as pallet_identity::Config>::MaxRegistrars,
+			<Runtime as pallet_identity::Config>::IdentityInformation,
+		>,
+		Option<BoundedVec<u8, <Runtime as pallet_identity::Config>::MaxUsernameLength>>,
+	);
+
+	/// Alias for `IdentityOf` from `pallet_identity`.
+	#[frame_support::storage_alias(pallet_name)]
+	pub(crate) type IdentityOf =
+		StorageMap<IdentityPallet, Twox64Concat, AccountId, Identity, OptionQuery>;
+
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		/// Clear requested judgements that do not have a corresponding deposit reserved.
+		///
+		/// This is successful only if the `target` account has judgements to clear. The transaction
+		/// fee is refunded to the caller if successful.
+		#[pallet::call_index(0)]
+		#[pallet::weight(Pallet::<T>::weight_clear_judgement())]
+		pub fn clear_judgement(
+			_origin: OriginFor<T>,
+			target: AccountId,
+		) -> DispatchResultWithPostInfo {
+			let identity = IdentityPallet::identity(&target).ok_or(Error::<T>::NotFound)?;
+			let (removed, identity) = Self::do_clear_judgement(&target, identity);
+			ensure!(removed > 0, Error::<T>::NotFound);
+
+			IdentityOf::insert(&target, identity);
+
+			Ok(Pays::No.into())
+		}
+	}
+
+	impl<T: Config> Pallet<T> {
+		fn do_clear_judgement(account_id: &AccountId, mut identity: Identity) -> (u32, Identity) {
+			// `subs_of`'s query kind is value option, if non subs the deposit is zero.
+			let (subs_deposit, _) = IdentityPallet::subs_of(account_id);
+			// deposit without deposits for judgement request.
+			let identity_deposit = identity.0.deposit.saturating_add(subs_deposit);
+			// total reserved balance.
+			let reserved = Balances::reserved_balance(account_id);
+			// expected deposit with judgement deposits.
+			let mut expected_total_deposit = identity_deposit;
+			// count before cleaning up the judgements.
+			let judgements_count = identity.0.judgements.len();
+
+			identity.0.judgements.retain(|(_, judgement)| {
+				if let Judgement::FeePaid(deposit) = judgement {
+					expected_total_deposit = expected_total_deposit.saturating_add(*deposit);
+					reserved >= expected_total_deposit
+				} else {
+					true
+				}
+			});
+
+			((judgements_count - identity.0.judgements.len()) as u32, identity)
+		}
+
+		/// Weight calculation for the worst-case scenario of `clear_judgement`.
+		/// Equal to 1 identity read/write + 1 subs read + 1 reserve balance read.
+		fn weight_clear_judgement() -> Weight {
+			<Runtime as frame_system::Config>::DbWeight::get().reads_writes(3, 1)
+		}
```

Comment:
We need a benchmark. This is a parachain and thus, we need to know the `pov_size`

Reply:
Updated with the benchmark function and the weight from the target machine